ssc/cmod_pvsamv1.cpp

```diff
@@ -1524,6 +1539,51 @@ void cm_pvsamv1::exec( ) throw (compute_module::general_error)
 	// The shared inverter of the PV array and a tightly-coupled DC connected battery
 	std::unique_ptr<SharedInverter> sharedInverter(new SharedInverter(inv_type, num_inverters, &snlinv, &plinv));
 
+	// Inverter thermal derate curves
```

Comment:
This section might also change if we make use of an SSC_MATRIX

Reply:
Done!